### DISCUSSION TITLE
"Page data from page-data.json for the failed page /" while building static HTML on Gatsby Cloud deploy

### DISCUSSION THREAD

**Discussion by mys31f** [gatsbyjs/gatsby#35335] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

I have a website that I've been trying to deploy to Gatsby Cloud and have always been getting this problem: 
![image](https://user-images.githubusercontent.com/59662061/161389067-7f861a8a-a784-41a0-86eb-3c318367aa36.png)

### Reproduction Link

https://github.com/mys31f/class-site

### Steps to Reproduce

1. Link a fork of this repo to a Gatsby Cloud site.
2. Let the site build automatically.

### Expected Result

The site builds normally.

### Actual Result

Site fails to build and ends at the error.

### Environment

```shell
I'm using Gatsby Cloud so, I'm not really sure.
```

**Comment by HelpMe-Pls** [comment]
I'm having the same issue for my rebuild of the deploy. My previous builds were fine, so I don't really know what's going there. The error message is pretty vague as well: 
![image](https://user-images.githubusercontent.com/43715386/163595043-dbe4ccba-a7b7-4498-b0f9-06f3303646ea.png)

Have you found the solution for this ?

**Reply by mys31f** [reply]
For this site, not yet. I had to make another site since I didn't want to bother with taking time to fix something that might take forever to solve.

~~thử mãi gần 1 tuần ko đc haiz~~

**Reply by HelpMe-Pls** [reply]
Lmao ikr ( ˘︹˘ )
Give up and move on thôi ông 🤷‍♂️
